mongo/integration/client_side_encryption_prose_test.go

```diff
@@ -40,6 +40,7 @@ const (
 	keySubtype               byte = 4                // expected subtype for data keys
 	encryptedValueSubtype    byte = 6                // expected subtypes for encrypted values
 	cryptMaxBsonObjSize           = 2097152          // max bytes in BSON object when auto encryption is enabled
```

Comment:
This in no longer an accurate name

Reply:
Renamed to `cryptMaxBatchSizeBytes` in the next commit.